package.json

```diff
@@ -26,5 +55,21 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "husky": "^4.2.5",
+    "lint-staged": "^10.2.11",
+    "prettier": "2.0.5"
+  },
```

Comment:
prettier has nothing to do with TypeScript

Reply:
Removing....